tests/integration/poster/poster_server_mock_test.py

```diff
@@ -0,0 +1,56 @@
+
+from tests.integration.poster.poster_server_mock import PosterServerMock
+
+import requests
+
+
+class TestPosterServerMock():
+    """ Tests for PosterServerMock. Tests the generic behaviors of
+    PosterServerMock.
+    """
+
+    def test_get_on_invalid_url(self):
+        server = PosterServerMock
+        server.start(server)
+        server.PATTERN=r"anything"
+        response = requests.get(
+            url="http://localhost:{port}/otherthing".format(
+                port=server.port
+            )
+        )
+        assert response.status_code == 404
+
+    def test_get_on_valid_url(self):
+        server = PosterServerMock
+        server.start(server)
+        server.PATTERN=r"anything"
+        response = requests.get(
+            url="http://localhost:{port}/anything".format(
+                port=server.port
+            )
+        )
+        assert response.content == b'"get content"' \
```

Comment:
@paulodamaso I think better do 
`assert response.content == json.dumps("get content")`
`assert response.status_code == HTTPStatus.OK`

Reply:
@vladarefiev I'd prefer to compare the raw value instead of taking the json dump, please take a look at the correction